pandas/tests/window/test_apply.py

```diff
@@ -138,3 +138,13 @@ def test_invalid_kwargs_nopython():
         Series(range(1)).rolling(1).apply(
             lambda x: x, kwargs={"a": 1}, engine="numba", raw=True
         )
+
+
+def test_apply_kwargs():
+    def foo(x, par):
```

Comment:
Could you add the Github issue number as a comment? `# GH ...`
Could you also test passing `args`?

Reply:
> Could you also test passing `args`?

+1